src/main/java/edu/kit/ipd/crowdcontrol/objectservice/moneytransfer/MoneyTransferManager.java

```diff
@@ -0,0 +1,262 @@
+
+    private void payWorker(WorkerRecord worker, List<GiftCodeRecord> giftCodes) throws MoneyTransferException {
+        if (!giftCodes.isEmpty()) {
+            StringBuilder paymentMessage = loadMessage("src/main/resources/PaymentMessage.txt");
+            StringBuilder giftCodeMessage = new StringBuilder();
+
+            //saves payment to the database
+            for (GiftCodeRecord rec : giftCodes) {
+                workerBalanceOperations.addDebit(worker.getIdWorker(), rec.getAmount(), rec.getIdGiftCode());
+                giftCodeMessage.append(rec.getCode()).append(System.getProperty("line.separator"));
+            }
+
+            //creates payment message
+            Map<String, String> map = new HashMap<>();
+            map.put("GiftCodes", giftCodeMessage.toString());
+            paymentMessage = new StringBuilder(Template.apply(paymentMessage.toString(), map));
+
+            //sends payment message
+            try {
+                mailHandler.sendMail(worker.getEmail(), "Your payment for your Crowdworking", paymentMessage.toString());
+            } catch (MessagingException e) {
+                throw new MoneyTransferException("The MailHandler couldnt send mails to crowdworkers." +
+                        "It seems, that there is either a problem with the server or with the properties file.");
+            } catch (UnsupportedEncodingException e) {
+                LOGGER.error(e);
+            }
+        }
+    }
+
+    private void sendNotification() throws MoneyTransferException {
+        StringBuilder message = new StringBuilder();
+        message.append("Dear administrator, ").append(System.getProperty("line.separator"));
+        message.append("we want to give you the following information:").append(System.getProperty("line.separator"));
+        message.append(notificationText);
+
+        try {
+            if (notificationText.length() != 0) {
+                LOGGER.trace("Started sending a notification about problems with submission of giftcodes.");
+                mailHandler.sendMail(notificationMailAddress, "Payment Notification", message.toString());
+                LOGGER.trace("Completed sending a notification about problems with submission of giftcodes.");
+            }
+        } catch (MessagingException e) {
+            throw new MoneyTransferException("The MailHandler couldnt send mails to the administrator." +
+                    "It seems, that there is either a problem with the server or with the properties file.");
+        } catch (UnsupportedEncodingException e) {
+            LOGGER.error(e);
+        }
+    }
+
+    private void sendErrorMessage(String errorMessage) {
+        StringBuilder message = new StringBuilder();
+        message.append("Dear administrator, ").append(System.getProperty("line.separator"));
+        message.append("it seems, that a critical error during payment occured:").append(System.getProperty("line.separator"));
+        message.append(errorMessage);
```

Comment:
Can you also move this message into a text file? So for example a translater could simply translate the message for the admin.